index.html

```diff
@@ -257,16 +288,18 @@ <h2>Other links</h2>
         <li><a href="https://www.opengeospatial.org/standards/">OGC® Standards and Supporting Documents</a>
           <p>
             The Open Geospatial Consortium (OGC) publishes standards for map-related data,
-            many of which are used or adapted for MapML.
+            some of which are used or adapted by MapML.
           </p>
         </li>
-        <li><a href="https://docs.opengeospatial.org/per/17-019.html">MapML Engineering Report (2017)</a> from the <abbr title="Open Geospatial Consortium">OGC</abbr> Testbed 13
+        <li>MapML Engineering Reports from the <abbr title="Open Geospatial Consortium">OGC</abbr> Innovation Program
```

Comment:
Don't need the `target="_blank"` for the links.

Also: Maybe move these all up to the end of the “Specifications and other reports” section, so they're not lost in “Other Links”?

Reply:
OK